modules/behavioural_qc/php/models/behaviouraldto.class.inc

```diff
@@ -0,0 +1,66 @@
+<?php declare(strict_types=1);
+/**
+ * PHP version 7
+ *
+ * @category Behavioural
+ * @package  Loris
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ *           Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+namespace LORIS\behavioural_qc\Models;
+
+/**
+ * A ProfileDTO represents a profile in the Behavioural Quality Control module.
+ * It is doing the mapping between the profile table columns
+ * and the JSON object properties returned to the frontend.
+ *
+ * Additionally, it implements the DataInstance interface so it can be used by
+ * a Database Provisioner.
+ *
+ * @category Behavioural
+ * @package  Loris
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ *           Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+class BehaviouralDTO implements \LORIS\Data\DataInstance
+{
+    /**
+     * Implements \LORIS\Data\DataInstance interface for this row.
+     *
+     * @return string
+     */
+    public function toJSON(): string
+    {
+        return json_encode(
+            array(
+                'FeedbackID'     => $this->FeedbackID,
+                'CandID'         => $this->CandID,
+                'SessionID'      => $this->SessionID,
+                'CommentID'      => $this->CommentID,
+                'Feedback_level' => $this->Feedback_level,
+                'Test_name'      => $this->Test_name,
+                'FieldName'      => $this->FieldName,
+                'Instrument'     => $this->Instrument,
+                'Visit_label'    => $this->Visit_label,
+                'PSCID'          => $this->PSCID,
+                'Project'        => $this->Project,
+                'Site'           => $this->Site,
+            )
+        );
+    }
+    /**
+     * Returns the PSCID for this row, for filters such as
+     * \LORIS\Data\Filters\UserSiteMatch to match again.
```

Comment:
UserSiteMatch uses `getCenterID(s?)`, not `getPSCID()`.

This is also missing the equivalent for project. It should probably include a getter for all of the fields provided.

Reply:
@driusan not being used. removed.